src/lambda/commands/invokeLambda.ts

```diff
@@ -123,41 +131,42 @@ export async function invokeLambda(
     }
 }
 
-function createMessageReceivedFunc(
+function createMessageReceivedFunc({fn, outputChannel, ...restParams}: {
+    // TODO: Consider passing lambdaClient: LambdaClient
+    fn: FunctionNodeBase, // TODO: Replace w/ invokeParams: {functionArn: string} // or Lambda.Types.InvocationRequest
+    outputChannel: vscode.OutputChannel
     resourceFetcher: ResourceFetcher,
     resourcePath: string,
-    postMessage: (message: any) => Thenable<boolean>,
-    fn: FunctionNodeBase
-) {
+    onPostMessage(message: any): Thenable<boolean>
```

Comment:
Out of curiosity, what are the criteria you're using to decide between `onFoo()` and just `foo()`?

Reply:
Good question. I like the idea of using convention to give insight into the purpose of a variable or param. Just like the `is` prefix gives the reader/coder the expectation that a variable expects boolean, the `on` prefix hints that a function is expected and that this is the opportunity for the developer to influence the behavior of the function. It's modeled after HTML DOM JavaScript's handler functions so it should be somewhat familiar to most readers.